src/actions/guides/emails/sending-emails-with-carbon.cr

```diff
@@ -7,11 +7,53 @@ class Guides::Emails::SendingEmailsWithCarbon < GuideAction
 
   def markdown : String
     <<-MD
-    ## Configuring Carbon
+    ## Carbon Configuration File
```

Comment:
Since these titles are indexed in to the search algolia, I wonder if this should actually say something like "Configuring Emails", and then have a section that explains what "Carbon" is. What are your thoughts? I would be good either way.

Reply:
That's a great idea, Jeremy! I'll push a change and see how it feels to read through.